README.md

```diff
@@ -1 +1,177 @@
-# angular-sharing-directive-controllers-readme
\ No newline at end of file
+# Sharing Directive Controllers
+
+## Overview
+
+Directives become even more powerful because we can communicate between their controllers. For instance, if we wanted to create a tabs component, we'd want to have our main `tabs` directive, and then directives for every tab. We can then link these together in order to create our tab picker.
+
+## Objectives
+
+- Describe the `require` property
+- Create another Directive
+- Create another Controller
+- Bind Controller to Directive
+- Import another Controller's instance
+
+## require
+
+We can require the parent controller for a certain directive by using the `require` property in our directive. We pass in a string, with the directives name that we want.
+
+Following on from our example above, we might have the following setup:
+
+```html
+<tabs>
+  <tab label="Tab 1">
+    Tab 1 contents!
+   </tab>
+   <tab label="Tab 2">
+    Tab 2 contents!
+   </tab>
+   <tab label="Tab 3">
+    Tab 3 contents!
+   </tab>
+</tabs>
+```
+
+Imagine our `tabs` directive looks like this:
+
+```js
+function tabs() {
+  return {
+    restrict: 'E',
+    scope: {},
+    transclude: true,
+    controller: function () {
+      this.tabs = [];
+    },
+    controllerAs: 'tabs',
+    template: [
+      '<div class="tabs">',
+        '<ul class="tabs__list"></ul>',
+        '<div class="tabs__content" ng-transclude></div>',
+      '</div>'
+    ].join('')
+  };
+}
+
+angular
+  .module('app', [])
+  .directive('tabs', tabs);
+```
+
+You might notice a property we haven't mentioned before - `transclude`. Don't worry about this yet, we're going to learn this very shortly!
```

Comment:
Transclude is required - our children directives are the ones that require the controller, and these are passed through. It wouldn't make sense for our tabs component to have hard coded tabs - this would remove the reusability of the directive.